pegasus/router.rb

```diff
@@ -349,6 +349,23 @@ def document(path)
       raise
     end
 
+    def render_partials(template_content)
+      # Template types that do not have thier own way of rendering partials
+      # (ie, markdown) can include other partials with the syntax:
+      #
+      #     {{ path/to/partial }}
+      #
+      # Because such content can be translated, we want to make sure that if a
+      # translator accidentally translates the path to the template, we simply
+      # render nothing rather than throwing an error
```

Comment:
It is worth tracking these errors somewhere (not sure how)? We could occasionally go through all of the invalid template paths and fix any caused by translations.

Reply:
Oh, I like that! We could probably register a honeybadger error here but still show the page. I'll add a Jira item for this.